ZoomZoom/Private/Get-DaysInRange.ps1

```diff
@@ -0,0 +1,48 @@
+<#
+.SYNOPSIS
+    Returns the days, months, and years of all days between 2 days
+.DESCRIPTION
+    Calculates a timespan between 2 dates, then returns each day between those 2 dates as datetime objects
```

Comment:
Is it inclusive or exclusive?

1-3 inclusive returns 1, 2, and 3
1-3 exclusive returns 2